### PR TITLE
Add AR preview for usdz files

### PR DESCRIPTION
**Pull request by shichongrui**
refs: MBL-12207
affects: Teacher
release note: Adds preview ability for usdz files

### CODE REVIEW

**Comment by inst-danger** [comment]
#### Release Note: 
Adds preview ability for usdz files

#### Affected Apps: Teacher

[MBL-12207](https://instructure.atlassian.net/browse/MBL-12207)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 93.97% | 93.98% | -0.02%
**Student** | 93.19% | 93.19% | 0%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.62% | 92.62% | 0%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: Canvas/TechDebt/Files and Folders/FileViewController.m [hunk 1]
```diff
@@ -302,7 +302,7 @@ - (UIView *)legacyFileMessageView {
 
 - (void)addOpenInARButton {
     UIButton *button = [UIButton new];
-    [button setTitle:NSLocalizedStringFromTableInBundle(@"Open in AR", nil, [NSBundle bundleForClass: self.class], nil) forState:UIControlStateNormal];
+    [button setTitle:NSLocalizedStringFromTableInBundle(@"Augment Reality", nil, [NSBundle bundleForClass: self.class], nil) forState:UIControlStateNormal];
```
Comment (armstrongnate) [inline Canvas/TechDebt/Files and Folders/FileViewController.m]: 😆 Is this really what they want the title to be?

File: CanvasCore/CanvasCore/React Native Modules/QLPreviewManager.m [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+#import <UIKit/UIKit.h>
+#import <CanvasCore/CanvasCore-Swift.h>
+
+@implementation QLPreviewManager
+
+RCT_EXPORT_MODULE();
+
+RCT_EXPORT_METHOD(previewFile:(NSString *)url)
+{
+    _fileURL = [[NSURL alloc] initWithString:url];
```
Comment (armstrongnate) [inline CanvasCore/CanvasCore/React Native Modules/QLPreviewManager.m]: This isn't bad but it's normally preferred to use `self.fileURL` in this kind of situation.

File: CanvasCore/CanvasCore/React Native Modules/QLPreviewManager.m [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+    return YES;
+}
+
+#pragma mark - QLPreviewControllerDataSource
+- (NSInteger)numberOfPreviewItemsInPreviewController:(QLPreviewController *)controller {
+    return 1;
+}
+
+- (id<QLPreviewItem>)previewController:(QLPreviewController *)controller previewItemAtIndex:(NSInteger)index {
+    return _fileURL;
```
Comment (armstrongnate) [inline CanvasCore/CanvasCore/React Native Modules/QLPreviewManager.m]: Same here.

File: Canvas/TechDebt/Files and Folders/FileViewController.m (hunk 1 above)
Comment (shichongrui) [reply Canvas/TechDebt/Files and Folders/FileViewController.m]: Haha yeah

**Pull request merged**